dotCMS/src/integration-test/java/com/dotmarketing/portlets/rules/RuleDataGen.java

```diff
@@ -30,6 +30,7 @@
 
     private Rule.FireOn fireOn = Rule.FireOn.EVERY_PAGE;
     private String name = "defaultName";
+    private Host host;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field host has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4883479)